core/src/components/fab-button/readme.md

```diff
@@ -4,6 +4,13 @@ Floating Action Buttons (FABs) represent the primary action in an application. B
 
 If the FAB button is not wrapped with `<ion-fab>`, it will scroll with the content. FAB buttons have a default size, a mini size and can accept different colors:
 
+The below chart has a description of each slot.
```

Comment:
The documentation of slots is part of stencil:
```
 * @slot - Content is placed between the named slots if provided without a slot.
```
add a comment like this in the component itself, like this: https://github.com/ionic-team/ionic/blob/master/core/src/components/toolbar/toolbar.tsx#L10-L14

Then run `stencil build` it will update the readme